app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md

```diff
@@ -0,0 +1,321 @@
+
+
+#### Scholarships for physics, French, German and Spanish
+
+To get a scholarship, you’ll need to be a high-quality student and pass a tough selection process, run by the professional body for your subject.
+
+If you’re successful, your teacher training provider will pay the scholarship money directly into your bank account over a period of 10 months.
+
+You may get a:
+
+* physics scholarship of £29,000
+* French, German or Spanish scholarship of £27,000
+
+To apply for a scholarship, you’ll need a place on a course training to teach French, German, Spanish or physics with an accredited teacher training provider in England, in the academic year 2023 to 2024.
+
+You’ll also need at least one of the following:
+
+* a bachelor’s degree class 2:1 (honours) or higher
+* a master’s degree
+* a PhD
+
+In exceptional circumstances, you may be offered a scholarship with a bachelor’s degree class 2:2, if you have significant relevant experience.
+You can check if you’re eligible for a scholarship and apply by visiting:
+
+* the [Institute of Physics](https://www.iop.org/about/support-grants/iop-teacher-training-scholarships#gref) (for a physics scholarship) 
+* the [British Council](https://www.britishcouncil.org/education/he-science/opportunities/ltts) (for a French, German or Spanish scholarship)
+
+[Find teacher training courses](https://www.gov.uk/find-postgraduate-teacher-training-courses) in languages and physics. 
+
+## Eligibility for other types of financial support
+
+Depending on your immigration status, you may be able to get other kinds of financial help from the UK government. 
+
+You are more likely to get financial support if you have some type of settled status in the UK – for example, through the EU Settlement Scheme or indefinite leave to remain. Without settled status, most non-UK candidates will not be eligible for financial support.
+
+Financial support is only available for courses where you pay a fee to study teacher training. 
+
+### Support for Irish citizens
+
+You will generally be eligible for a tuition fee loan and a student maintenance loan if you meet both of the following requirements:
+
+* you are an Irish citizen
+* you have been living in the UK, Guernsey, Jersey or the Isle of Man throughout the 3 years immediately before the start of your course
+
+You will generally be eligible for a tuition fee loan if you meet both the following requirements:
+
+* you are an Irish citizen
+* you have been living in Ireland, or Ireland and the UK, Guernsey, Jersey or the Isle of Man, throughout the 3 years before the start of your course
+
+### Support for trainee teachers  from the European Economic Area (EEA) or Switzerland
+
+If you meet both the following requirements:
+
+* you have settled status under the EU Settlement Scheme
+* you have been resident in the UK throughout the 3 years before the start of your course
+
+you will generally be eligible for a:
```

Comment:
I wonder if this whole section is clear enough? Might there be a better way to separate this info out using subheadings somehow?